yorkie/src/main/kotlin/dev/yorkie/document/change/ChangePack.kt

```diff
@@ -0,0 +1,33 @@
+package dev.yorkie.document.change
+
+import dev.yorkie.document.time.TimeTicket
+
+/**
+ * [ChangePack] is a unit for delivering changes in a document to the remote.
+ *
+ * @property documentKey the key of the document.
+ * @property checkPoint It is used to determine the client received changes.
+ * @property snapshot a byte array that encodes the document.
+ * @property minSyncedTicket the minimum logical time taken by clients who attach to the document.
+ * It is used to collect garbage on the replica on the client.
+ */
+internal class ChangePack(
+    val documentKey: String,
+    val checkPoint: CheckPoint,
+    val changes: List<Change>,
+    val snapshot: List<UByte>?,
```

Comment:
used `List<UByte>` to represent `Uint8Array` in JS.
is it ok to do so?
https://kotlinlang.org/docs/unsigned-integer-types.html

Reply:
`snapshot` defined in protobuf model is compiled into ByteString by java protobuf, so I think it is better use ByteString as is.
since ByteString is Iterable of Byte, converting to List is simple but why make an extra step by converting back and forth unless it is really necessary :)